docs/getting-started/locally/binary.md

```diff
@@ -0,0 +1,21 @@
+# Install Binary
+
+To run the nio Platform locally, you need a version of a nio [binary](/binaries).
+
+>**[info] Requirements**
+>
```

Comment:
add the requirement: Python 3.4 or greater

Reply:
Oh, I see, maybe start the bullets with the requirement, then follow with the action to take.